### PR TITLE
to have a unique tagname for all robots, to have a integration data

### PR DESCRIPTION
**Pull request by egolesor**
To add a new parser tags: 
In the file BookTagUtil add the annotation
1. `@PropertySource({"classpath:/ebooks.properties", "YOUR_TAGS"})`
2. Add your library into `model.Library` 
3. Add into the BookTagUtil::tagNamesOf your library

*\* use this util into your robot

### CODE REVIEW

File: RobotCrawler/src/main/java/robot/ebooks/BookTagUtil.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            eBooksTagMapper.put(bookTag,
+                    Arrays.asList(environment.getProperty(bookTag.name()).split(",")));
+        }
+    }
+
+    /**
+     * To get the specific tag name of {@link BookTag} from the {@link Library}
+     * @param bookTag
+     * @param library
+     * @return
```
Comment (caroo7) [inline RobotCrawler/src/main/java/robot/ebooks/BookTagUtil.java]: Maybe more specific javadosc? Info about params and return value.

File: RobotCrawler/src/main/java/server/service/BookFinderController.java [hunk 2]
```diff
@@ -5,32 +5,40 @@
...
-        ResponseBase<EnumMap<EBookCategory, List<BookData>>> response = new EBooksResponse();
+        ResponseBase<EnumMap<BookTag, List<BookData>>> response = new EBooksResponse();
         response.setResult(robot.getOffers(categories));
-        return response;
+        return null;
     }
 
+    @ResponseBody
+    @GetMapping("/kkkk")
+    public EnumMap<BookTag, List<BookData>> kkk(){
```
Comment (caroo7) [inline RobotCrawler/src/main/java/server/service/BookFinderController.java]: Nice name :+1:  I'm wondering why method name is 'kkk' but maping is 'kkkk'? :D

File: RobotCrawler/src/test/java/robot/ebooks/EBooksRobotManagerTest.java [hunk 3]
```diff
@@ -1,16 +1,12 @@
 package robot.ebooks;
 
-import model.EBookCategory;
-import org.testng.annotations.Test;
-import org.testng.asserts.SoftAssert;
-
 public class EBooksRobotManagerTest {
 
-    @Test
+    /*@Test
```
Comment (caroo7) [inline RobotCrawler/src/test/java/robot/ebooks/EBooksRobotManagerTest.java]: Remove test if you don't need it.

File: RobotCrawler/src/main/java/robot/ebooks/BookTagUtil.java (hunk 1 above)
Comment (egolesor) [reply RobotCrawler/src/main/java/robot/ebooks/BookTagUtil.java]: Done

File: RobotCrawler/src/test/java/robot/ebooks/EBooksRobotManagerTest.java (hunk 3 above)
Comment (egolesor) [reply RobotCrawler/src/test/java/robot/ebooks/EBooksRobotManagerTest.java]: Done

**Comment by LukaszCzubernat** [comment]
:+1:

**Pull request closed without merging**